lib/builderator/config/file.rb

```diff
@@ -0,0 +1,210 @@
+require 'time'
+
+require_relative './attributes'
+require_relative '../util'
+
+module Builderator
+  module Config
+    ##
+    # DSL Loader for a configuration file
+    ##
+    class File < Attributes
+      class << self
+        ## DSL Loaders
+        def from_file(source, **options)
```

Comment:
I feel like naming this `from_ruby` and changing they `:type` to `:ruby` would make be more in line with idea of loading configs from multiple types of files.